WordPress/src/main/java/org/wordpress/android/ui/domains/DomainRegistrationResultFragment.kt

```diff
@@ -41,6 +43,7 @@ class DomainRegistrationResultFragment : Fragment() {
         checkNotNull((activity?.application as WordPress).component())
 
         continue_button.setOnClickListener {
+            activity!!.setResult(RESULT_OK, Intent())
```

Comment:
Wonder, why are you passing empty `Intent()` to `setResult()` here?

Reply:
Hmmmm. Thats a good question. Not sure if I have a good answer. For some reason I thought the second argument on `setResult()` was required. Not sure why I was thinking that. I have removed the empty `Intent()`.